substratevm/src/com.oracle.svm.core.posix/src/com/oracle/svm/core/posix/JavaUtilZipSubstitutions.java

```diff
@@ -108,28 +109,29 @@ private static int updateByteBuffer(int crc, long addr, int off, int len) {
 }
 
 @Platforms({Platform.LINUX.class, Platform.DARWIN.class})
-@TargetClass(java.util.zip.Deflater.class)
+@TargetClass(value = java.util.zip.Deflater.class)
 final class Target_java_util_zip_Deflater {
-
     @Alias //
-    @TargetElement(onlyWith = JDK8OrEarlier.class) //
+    @TargetElement(onlyWith = JDK10OrEarlier.class) //
     private byte[] buf;
-
     @Alias //
-    @TargetElement(onlyWith = JDK8OrEarlier.class) //
+    @TargetElement(onlyWith = JDK10OrEarlier.class) //
     int off;
-
     @Alias //
-    @TargetElement(onlyWith = JDK8OrEarlier.class) //
+    @TargetElement(onlyWith = JDK10OrEarlier.class) //
     int len;
+    @Alias //
```

Comment:
I like the consistent formatting separating annotations from declarations.  I think there should be consistent use of whitespace: each (now) multi-line declaration should have whitespace around it.